daemon/mgr/container_checkpoint.go

```diff
@@ -0,0 +1,153 @@
+package mgr
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/alibaba/pouch/apis/types"
+)
+
+var (
+	checkpointConfigPath             = "config.json"
+	checkpointConfigPerm os.FileMode = 0700
+)
+
+// getCheckpointDir gets container checkpoint directory.
+func (mgr *ContainerManager) getCheckpointDir(container, prefixDir, checkpointID string, create bool) (string, error) {
```

Comment:
In the implementation of this function, I am wondering if we need to add more dedicated error instance when failed. Since these errors in `errtypes` packages could be used to construct the status codes for API layers. WDYT?